README.md

```diff
@@ -1,5 +1,3 @@
 # replays
 Repository of replays for OpenRCT2. Used for testing.
 
-### Build Status
-[![Build status](https://ci.appveyor.com/api/projects/status/oimpt68xif91vlyx/branch/master?svg=true)](https://ci.appveyor.com/project/OpenRCT2/replays/branch/master)
```

Comment:
I think this could have been kept, if desired: [GitHub Docs | Adding a workflow status badge
](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge)

Reply:
I don't think it really makes sense it was always pretty pointless for replays